hs_core/views/resource_folder_hierarchy.py

```diff
@@ -334,6 +339,16 @@ def data_store_folder_unzip_public(request, pk, pathname):
     return data_store_folder_unzip(request, res_id=pk, zip_with_rel_path=pathname)
 
 
+@api_view(['POST'])
+def ingest_metadata_files(request, pk):
+    resource, _, _ = view_utils.authorize(request, pk,
+                                          needed_permission=ACTION_TO_AUTHORIZE.EDIT_RESOURCE)
+    resource_files = list(request.FILES.values())
+    from hs_file_types.utils import identify_and_ingest_metadata_files
+    identify_and_ingest_metadata_files(resource, resource_files)
```

Comment:
It'd be good to move the import to the beginning of the file or beginning of the method.

Reply:
done

Reply:
In the method avoids import loops. Great!